rskj-core/src/test/java/co/rsk/rpc/netty/RskWebSocketJsonRpcHandlerTest.java

```diff
@@ -50,15 +50,15 @@
     public void setUp() {
         emitter = mock(EthSubscriptionNotificationEmitter.class);
         serializer = mock(JsonRpcSerializer.class);
-        handler = new RskWebSocketJsonRpcHandler(emitter, serializer);
+        handler = new RskWebSocketJsonRpcHandler(emitter);
     }
 
     @Test
     public void visitUnsubscribe() {
         EthUnsubscribeRequest unsubscribe = new EthUnsubscribeRequest(
                 JsonRpcVersion.V2_0,
                 RskJsonRpcMethod.ETH_UNSUBSCRIBE,
-                35,
+                "35",
```

Comment:
I would not change this test and would leave it as it was and maybe we can add another if necessary.